pennylane/_qubit_device.py

```diff
@@ -895,7 +889,7 @@ def adjoint_jacobian(self, tape, starting_state=None, use_device_state=False):
         dot_product_real = lambda b, k: self._real(qmlsum(self._conj(b) * k, axis=sum_axes))
 
         for m in tape.measurements:
-            if m.return_type is not qml.operation.Expectation:
+            if m.return_type is not qml.measurements.Expectation:
```

Comment:
I guess we don't need to be explicit here:
```suggestion
            if m.return_type is not Expectation:
```